presto-spi/src/main/java/com/facebook/presto/spi/relation/LogicalRowExpressions.java

```diff
@@ -275,4 +327,25 @@ else if (!seen.contains(expression)) {
 
         return Collections.unmodifiableList(result);
     }
+
+    private boolean isNotCallExpression(RowExpression expression)
```

Comment:
What about `isNegationExpression`? I first understand it as "is not a call expression" and didn't understand...